src/pages/markets/components/markets/_basket-indices.tsx

```diff
@@ -51,6 +54,34 @@ const BasketIndices = ({ simple_step_content }: BasketIndicesProps) => {
                             <Localize translate_text="Baskets trades  available on Deriv" />
                         }
                     />
+                    <Flex.Box
+                        direction="col"
+                        container="fluid"
+                        justify="center"
+                        align="center"
+                        pb="10x"
+                        md={{ pb: '40x', mb: '20x' }}
+                    >
+                        <Typography.Paragraph
+                            size="medium"
+                            mb="10x"
+                            textcolor="black"
+                            align="center"
+                        >
+                            <Localize translate_text="_t_Want to know more about CFD trading conditions for the instruments we offer?_t_" />
+                        </Typography.Paragraph>
+                        <LinkButton.Primary
+                            font_family="UBUNTU"
```

Comment:
now this font-family is global, you can try to remove this props and check that font-family still the same

Reply:
Just tried removing that and font-family turns to IBM Plex Sans. Maybe I have to keep this prop for now

Reply:
Agree, thanks for checking this